### PR TITLE
Add fetching shipping methods

### PR DESCRIPTION
**Pull request by danhaj**

### CODE REVIEW

File: packages/theme/components/Checkout/VsfShippingProvider.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  setup() {
+    const { $spree } = useVSFContext();
+    const selectedMethod = ref(null);
+    const shippingMethods = ref([]);
+
+    const selectMethod = method => selectedMethod.value = method;
+
+    onMounted(async () => {
+      const { shippingIds, methods } = await $spree.api.getShippingMethods();
+      console.log(shippingIds);
```
Comment (rafalcymerys) [inline packages/theme/components/Checkout/VsfShippingProvider.vue:61]: This seems like a leftover 🙂

File: packages/theme/components/Checkout/VsfShippingProvider.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
+    <p>
+      <b>Please implement vendor specific VsfShippingProvider component in 'components/Checkout' directory</b>
```
Comment (rafalcymerys) [inline packages/theme/components/Checkout/VsfShippingProvider.vue:4]: I think we could remove this message

**Comment by rafalcymerys** [comment]
How is the data passed back to the checkout? I think we're missing that part. 

Also, getShippingMethods can return multiple shipments. User should be able to select a different shipping method for each of them (this is sometimes the case when some products can be shipped via express delivery and some cannot, Spree will then split the order into multiple shipments). Do you think that's something we could also handle?

File: packages/theme/components/Checkout/VsfShippingProvider.vue (hunk 2 above)
Comment (diegolezana) [reply packages/theme/components/Checkout/VsfShippingProvider.vue]: Done

File: packages/theme/components/Checkout/VsfShippingProvider.vue (hunk 1 above)
Comment (diegolezana) [reply packages/theme/components/Checkout/VsfShippingProvider.vue]: Done

**Comment by diegolezana** [comment]
> How is the data passed back to the checkout? I think we're missing that part.
> 
> Also, getShippingMethods can return multiple shipments. User should be able to select a different shipping method for each of them (this is sometimes the case when some products can be shipped via express delivery and some cannot, Spree will then split the order into multiple shipments). Do you think that's something we could also handle?

I modified the code to make it similar as yuty project. I assume this will fix the issue, however I still need to confirm this, there are a specific way to test it? At the moment I only tested changing single shipment method and then use postman to see if works and for now it is fine.

**Pull request merged**